ThingIFSDK/ThingIFSDK/CommandForm.swift

```diff
@@ -64,20 +64,39 @@ open class CommandForm<ConcreteAlias: Alias>: NSObject, NSCoding {
     }
 
     open func encode(with aCoder: NSCoder) {
-        aCoder.encode(self.actions, forKey: "actions")
+        var dictArray: [[String : Any]] = []
+        self.actions.forEach {
+            action in dictArray.append(
```

Comment:
dictionary for traitActions and nonTraitActions are different. 

nonTraitActions: 
```js
{
   "actions": [
      {“<actionName>”:<Object|Array|Boolean|String|Number>}
   ],
   "title": "title of command",
   "description": "description of command"
}
```

traitActions: 
```js
{
  "actions": [
    {
       "<traitAlias>":[
         {“<actionName>”:<Object|Array|Boolean|String|Number>}
       ]
     }
  ],
   "title": "title of command",
   "description": "description of command"
}
```